File: pgm/ABC_Java/ABC_438_A.java
import java.io.*;
// import java.math.*;
// import java.time.*;
// import java.util.*;
// import java.util.Map.*;
// import java.util.stream.*;

public class Main {
    public static void main (String[] args) throws Exception {
        BufferedReader reader =
            new BufferedReader (new InputStreamReader (System.in));
        String[] arr = reader.readLine().split (" ");
        reader.close();
        final int one = 1;
        final int seven = 7;
        final int zero = 0;
        int D = Integer.parseInt (arr[zero]);
        int F = Integer.parseInt (arr[one]);
        System.out.println ((F + seven * ((D - F) / seven + one) - one) % D +
                            one);
        System.exit (0);
    }
}

File: pgm/ABC_C#/ABC_438_A.cs
using System;
// using System.Collections.Generic;
// using System.Globalization;
// using System.IO;
// using System.Linq;
// using System.Text;
// using System.Text.RegularExpressions;

class Program
{
    static void Main()
    {
        string[] arr = Console.ReadLine().Split(' ');
        const int one = 1;
        const int seven = 7;
        const int zero = 0;
        int D = int.Parse(arr[zero]);
        int F = int.Parse(arr[one]);
        Console.WriteLine(
            (F + seven * ((D - F) / seven + one) - one) % D + one);
        Environment.Exit(0);
    }
}